nemo/collections/asr/parts/preprocessing/features.py

```diff
@@ -47,6 +47,8 @@
 
 
 def normalize_batch(x, seq_len, normalize_type):
```

Comment:
This function may be used outside ASR, in TTS as well. Is it absolutely necessary to modify? If so we need a partial ok from TTS team.

Reply:
This function is just called one in the whole nemo and does not change the output, just added new outputs. Would check it out with someone from TTS team to make sure.